salt/states/virtualenv_mod.py

```diff
@@ -46,34 +46,36 @@ def managed(name,
             pip_download_cache=None,
             pip_exists_action=None,
             proxy=None,
+            pip_upgrade=False,
```

Comment:
@anlutro, will you move this kwarg to the end of the arg list to minimize API impact?  Thanks.